app/views/projects/show.html.erb

```diff
@@ -111,9 +116,28 @@
 
             <div class='row iteration <%= iteration.status %> '>
               <div class='card w-full'>
-                <h3><%= link_to iteration.title, [@project, iteration], class: 'underline hover:opacity-75' %> </h3>
+                <h3><%= link_to iteration.title, [@project, iteration], class: 'underline hover:opacity-75' %></h3>
+                <p class="mb-6"><%= iteration.description %></p>
+                <p class="mb-6"><%= pluralize(iteration.outcomes.count, 'outcome') %></p>
+                <h3>Latest update</h3>
+                <% if iteration.warning == :check_in_due %>
```

Comment:
Suggest extracting these conditionals into a helper module, class or method for easier testing and reducing complexity of views. Also the styling looks slightly different to the spec in some areas.